flume-shared/flume-shared-kafka-test/src/main/java/org/apache/flume/shared/kafka/test/KafkaPartitionTestUtil.java

```diff
@@ -199,7 +200,9 @@ public static void checkResultsAgainstSkew(PartitionTestScenario scenario,
 
       TopicPartition partition = new TopicPartition(topic, i);
 
-      consumer.assign(Arrays.asList(partition));
```

Comment:
What does this change bring?

Reply:
`KafkaConsume.assign` signature has been changed in 0.10.0.0
0.9.0.1 => `public void assign(java.util.List<org.apache.kafka.common.TopicPartition>);`
0.10.0.0 => `public void assign(java.util.Collection<org.apache.kafka.common.TopicPartition>);`

Reply:
+1